knowrob_srdl/knowrob_srdl/build.gradle

```diff
@@ -19,7 +19,7 @@ apply plugin: 'application'
 mainClassName = "org.knowrob.knowrob_srdl.URDF2SRDL"
 
 dependencies {
-  compile 'org.ros.rosjava_core:rosjava:[0.1,)'
+  compile 'org.ros.rosjava_core:rosjava:[0.3,0.4)'
```

Comment:
I wonder a bit, how do you come up with these version numbers?

Reply:
By locate rosjava_core on my machine, I found rosjava core 0.3.5
You can either specify the exact version [0.3.5], or allow everything >= 0.3 (using [0.3.+]), or allow version numbers between 0.3 and 0.4.
I was going for the latter, because I expect minor version changes to be not api breaking. Could also be set to 0.3.5 if you prefer.

Reply:
sounds reasonable